### PR TITLE
Replace InsightCard by Card component

### PR DESCRIPTION
**Pull request by pinotalexandre**
## Description
Replace InsightCard component with Sparkle Card component in Analytics.tsx.
- Use of the new Card component
- Maintain same structure and functionality
- Improve visual consistency with design system

## Risk
Minimal - UI changes

## Deploy Plan
Deploy front

### CODE REVIEW

**Comment by aubin-tchoi** [comment]
your lint did not pass @pinotalexandre, did you maybe forget to bump the version of sparkles used?

File: front/components/workspace/Analytics.tsx [hunk 1]
```diff
@@ -50,32 +22,66 @@ export function QuickInsights({ owner }: QuickInsightsProps) {
...
-          <InsightCard
+          <Card
             title="Members"
             subtitle="Total members"
-            value1={analytics.memberCount}
+            content={
+              <div className="text-lg font-semibold text-element-900">
+                {analytics.memberCount}
+              </div>
+            }
```
Comment (aubin-tchoi) [inline front/components/workspace/Analytics.tsx:32]: OOC: I would have rather passed this as a children personally

File: front/components/workspace/ProviderManagementModal.tsx [hunk 2]
```diff
@@ -37,7 +37,7 @@ const prettyfiedProviderNames: { [key in ModelProviderIdType]: string } = {
 const modelProviders: Record<ModelProviderIdType, string[]> =
   USED_MODEL_CONFIGS.reduce(
     (acc, model) => {
-      if (!model.isLegacy) {
+      if (model && !model.isLegacy) {
```
Comment (aubin-tchoi) [inline front/components/workspace/ProviderManagementModal.tsx:40]: FMI: how come do you need that?

Comment (pinotalexandre) [reply front/components/workspace/ProviderManagementModal.tsx:40]: For some reasons, I couldn't load the workspace page, I had to add this line to make it work (suggested by @q)

Comment (aubin-tchoi) [reply front/components/workspace/ProviderManagementModal.tsx:40]: can you try again without now that we have `types` running?

**Comment by aubin-tchoi** [comment]
meta: if you have screenshots to get a before/after you should add them to the description

**Review by aubin-tchoi — APPROVED** [review]
LGTM

**Review by JulesBelveze — COMMENTED** [review]
LGTM

**Pull request merged**